pkg/scheduler/flavorassigner/flavorassigner_test.go

```diff
@@ -2352,15 +1948,39 @@ func TestAssignFlavors(t *testing.T) {
 					ReclaimablePods: tc.wlReclaimablePods,
 				},
 			})
-			if tc.clusterQueue.FlavorFungibility.WhenCanBorrow == "" {
-				tc.clusterQueue.FlavorFungibility.WhenCanBorrow = kueue.Borrow
+			if tc.clusterQueue.Spec.FlavorFungibility.WhenCanBorrow == "" {
+				tc.clusterQueue.Spec.FlavorFungibility.WhenCanBorrow = defaultFlavorFungibility.WhenCanBorrow
 			}
-			if tc.clusterQueue.FlavorFungibility.WhenCanPreempt == "" {
-				tc.clusterQueue.FlavorFungibility.WhenCanPreempt = kueue.TryNextFlavor
+
+			if tc.clusterQueue.Spec.FlavorFungibility.WhenCanPreempt == "" {
+				tc.clusterQueue.Spec.FlavorFungibility.WhenCanPreempt = defaultFlavorFungibility.WhenCanPreempt
+			}
+
+			cache := cache.New(utiltesting.NewFakeClient())
+			if err := cache.AddClusterQueue(context.Background(), &tc.clusterQueue); err != nil {
+				t.Fatalf("failed to add CQ to cache")
 			}
-			tc.clusterQueue.UpdateWithFlavors(resourceFlavors)
-			tc.clusterQueue.UpdateRGByResource()
-			flvAssigner := New(wlInfo, &tc.clusterQueue, resourceFlavors, tc.enableFairSharing)
+			for _, rf := range resourceFlavors {
+				cache.AddOrUpdateResourceFlavor(rf)
+			}
+
+			clusterQueue := cache.Snapshot().ClusterQueues[tc.clusterQueue.Name]
+
+			if clusterQueue == nil {
+				t.Fatalf("failed to create CQ snapshot")
+			}
+
+			if tc.cohortResources != nil {
+				if clusterQueue.Cohort == nil {
+					t.Fatalf("test case has cohort resources, but cluster queue doesn't have cohort")
```

Comment:
I would begin the failure messages with capital letter as in the other places

Reply:
FYI we have https://google.github.io/styleguide/go/decisions.html#error-strings, otoh being consistent is good, I'm ok either way.

Reply:
"On the other hand, the style for the full displayed message (logging, **test failure**, API response, or other UI) depends, but should typically be capitalized."

So seems that capital is preferred in this case.